docs/knowledgebase/advanced/consensus.md

```diff
@@ -71,18 +74,17 @@ resolved by choosing the branch that has the most blocks built on it recursively
 
 ![GHOST rule](assets/consensus-ghost.png)
 
-## Block Production
+## Block Authoring
 
 Some nodes in a blockchain network are able to produce new blocks, a process known as authoring.
 Exactly which nodes may author blocks depends on which consensus engine you're using. In a
 centralized network, a single node might author all the blocks, whereas in a completely
-permissionless network, an algorithm must select the block author at each height.
+permissionless network, an algorithm must select the block author at each height. This iterative process confirms the integrity of the previous block.
 
 ### Proof of Work
 
-In proof-of-work systems like Bitcoin, any node may produce a block at any time, so long as it has
-solved a computationally-intensive problem. Solving the problem takes CPU time, and thus miners can
-only produce blocks in proportion with their computing resources. Substrate provides a proof-of-work
+In proof-of-work (PoW) systems like Bitcoin, any node may produce a block at any time, so long as it has
+solved a computationally-intensive problem, this creates a consensus mechanism. By utilizing proof-of-work, the node deters denial-of-service attacks and other service abuses by requiring some work from the service requester, usually this translates into processer time. Solving the problems take CPU time, and thus miners only produce blocks in proportion with their computing resources. Substrate provides a proof-of-work
```

Comment:
First , there is a comma splice where you first started editing.

Second, the PoW is for the right to author a block, not the right to query a node. The PoW here doesn't provide DoS protection. It differs from hashcash in this way.

Reply:
Yeah, revert this change. Besides what Joshy said, PoW does not create a consensus mechanism.